ext/MitsubaLoader/CElementMatrix.cpp

```diff
@@ -0,0 +1,83 @@
+#include "CElementMatrix.h"
+
+#include "ParserUtil.h"
+#include "CSimpleElement.h"
+
+namespace irr { namespace ext { namespace MitsubaLoader {
+
+
+bool CElementMatrix::processAttributes(const char** _atts)
+{
+
+	switch (type)
+	{
+	case CElementMatrix::Type::ARBITRARY:
+		for (int i = 0; _atts[i]; i += 2)
+		{
+			if (!std::strcmp(_atts[i], "value"))
+			{
+				std::pair<bool, core::matrix4SIMD> result = getMatrixFromString(_atts[i + 1]);
+				matrix = result.second;
+				return result.first;
+			}
+			else
+			{
+				ParserLog::wrongAttribute(_atts[i], getLogName());
+				false;
+			}
+		}
+	case CElementMatrix::Type::TRANSLATION:
```

Comment:
explain?

Reply:
it is not implemented yet

Reply:
sorry, I should leave some comment about that

Reply:
but still, explain?